coffeecard/CoffeeCard.MobilePay/Service/v2/MobilePayPaymentsService.cs

```diff
@@ -11,6 +11,8 @@ namespace CoffeeCard.MobilePay.Service.v2
 {
     public class MobilePayPaymentsService : IMobilePayPaymentsService
     {
+        private const string CoffeeCardAppRedirectUrl = "analogcoffeecard://mobilepay_purchase";
```

Comment:
Should this not be set by the config file, so we can redirrect to test app when in development?